jsonrpc/debug_handler.go

```diff
@@ -0,0 +1,179 @@
+package jsonrpc
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"runtime/pprof"
+	"runtime/trace"
+	"strings"
+	"sync"
+
+	"github.com/hashicorp/go-bexpr"
+)
+
+var (
+	errCPUProfilingInProgress    = errors.New("CPU profiling already in progress")
+	errCPUProfilingNotInProgress = errors.New("CPU profiling not in progress")
+	errTraceAlreadyInProgress    = errors.New("trace already in progress")
+	errTraceNotInProgress        = errors.New("trace not in progress")
+	identifierRegex              = regexp.MustCompile(`[:/\.A-Za-z0-9_-]+`)
+	notRegex                     = regexp.MustCompile(`!([:/\.A-Za-z0-9_-]+)`)
```

Comment:
This name confuses me... :smile:

Reply:
expressionRegex              = regexp.MustCompile(`[:/\.A-Za-z0-9_-]+`)
notExpressionRegex           = regexp.MustCompile(`!([:/\.A-Za-z0-9_-]+)`)

Is it better this way :)?